public/app/core/components/TimePicker/TimePickerWithHistory.tsx

```diff
@@ -34,6 +36,12 @@ export const TimePickerWithHistory = (props: Props) => {
               onAppendToHistory(value, values, onSaveToStore);
               props.onChange(value);
             }}
+            onError={(error?: string) =>
+              appEvents.emit(AppEvents.alertError, [
+                t('time-picker.copy-paste.default-error.title', 'Invalid time range'),
+                `"${error}" ${t('time-picker.copy-paste.default-error.message', 'is not a valid time range')}`,
```

Comment:
2 things here:
- [we use a maximum of 3 segments for the keys](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#phrase-id-naming-convention)
- you should pass the error to the translation as a variable so the translators have as much context as possible for how to translate the string. look up how e.g. `expand-folder-button` works